lib/exprotobuf.ex

```diff
@@ -8,44 +8,44 @@ defmodule Protobuf do
   alias Protobuf.Utils
 
   defmacro __using__(opts) do
-    namespace = __CALLER__.module
```

Comment:
This section of code could definitely use a refactoring. For now I was able to keep the diff fairly small, I'm just doing two nested `case` macros instead of a single `case` macro, but I think I would like to come back to cleanup how we are building the `%Config{}` struct here.